<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KM File Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="left_path">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="right_path">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableView" name="tableView_1"/>
    </item>
    <item row="2" column="1">
     <widget class="QTableView" name="tableView_2"/>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="renameButon">
        <property name="text">
         <string>F2 Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>F3 Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewButton">
        <property name="text">
         <string>F4 View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="text">
         <string>F5 Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveButton">
        <property name="text">
         <string>F6 Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newFileButton">
        <property name="text">
         <string>F7 New File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newDirButton">
        <property name="text">
         <string>F8 New Dir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>F9 Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>F10 Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="available_storage">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_file_manager"/>
    <addaction name="actionTeam"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionDefault_Zoom"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionNew_Folder"/>
    <addaction name="actionZip"/>
    <addaction name="actionExtract"/>
    <addaction name="actionExtract_to"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuRename">
     <property name="title">
      <string>Rename</string>
     </property>
     <addaction name="actionRename_File"/>
     <addaction name="actionRename_Directory"/>
    </widget>
    <widget class="QMenu" name="menuDelete">
     <property name="title">
      <string>Delete</string>
     </property>
     <addaction name="actionDelete_File"/>
     <addaction name="actionDelete_Directory"/>
    </widget>
    <widget class="QMenu" name="menuCopy">
     <property name="title">
      <string>Copy</string>
     </property>
     <addaction name="actionCopy_File"/>
     <addaction name="actionCopy_Directory"/>
    </widget>
    <widget class="QMenu" name="menuMove">
     <property name="title">
      <string>Move</string>
     </property>
     <addaction name="actionMove_File"/>
     <addaction name="actionMove_Directory"/>
    </widget>
    <addaction name="menuRename"/>
    <addaction name="menuDelete"/>
    <addaction name="menuCopy"/>
    <addaction name="menuMove"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionShow_in_Terminal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normalon>icons/Zoom In_32x32.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normaloff>icons/Zoom Out_32x32.png</normaloff>icons/Zoom Out_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="icon">
    <iconset>
     <normaloff>icons/New_32x32.png</normaloff>icons/New_32x32.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="icon">
    <iconset>
     <normaloff>icons/Folder_32x32.png</normaloff>icons/Folder_32x32.png</iconset>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionZip">
   <property name="icon">
    <iconset>
     <normaloff>icons/Archive_32x32.png</normaloff>icons/Archive_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Zip</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/Add_32x32.png</normaloff>icons/Add_32x32.png</iconset>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icons/Cancel_32x32.png</normaloff>icons/Cancel_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset>
     <normaloff>icons/Search_32x32.png</normaloff>icons/Search_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionShow_in_Terminal">
   <property name="text">
    <string>Show in Terminal</string>
   </property>
  </action>
  <action name="actionAbout_file_manager">
   <property name="icon">
    <iconset>
     <normaloff>icons/Help_32x32.png</normaloff>icons/Help_32x32.png</iconset>
   </property>
   <property name="text">
    <string>About file manager</string>
   </property>
  </action>
  <action name="actionTeam">
   <property name="icon">
    <iconset>
     <normaloff>icons/Information_32x32.png</normaloff>icons/Information_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Team</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>icons/Information_32x32.png</normaloff>icons/Information_32x32.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDefault_Zoom">
   <property name="icon">
    <iconset>
     <normaloff>icons/Cancel_32x32.png</normaloff>icons/Cancel_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Default Zoom</string>
   </property>
  </action>
  <action name="actionRename_File">
   <property name="text">
    <string>Rename File</string>
   </property>
  </action>
  <action name="actionRename_Directory">
   <property name="text">
    <string>Rename Directory</string>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="text">
    <string>Delete File</string>
   </property>
  </action>
  <action name="actionDelete_Directory">
   <property name="text">
    <string>Delete Directory</string>
   </property>
  </action>
  <action name="actionCopy_File">
   <property name="text">
    <string>Copy File</string>
   </property>
  </action>
  <action name="actionCopy_Directory">
   <property name="text">
    <string>Copy Directory</string>
   </property>
  </action>
  <action name="actionMove_File">
   <property name="text">
    <string>Move File</string>
   </property>
  </action>
  <action name="actionMove_Directory">
   <property name="text">
    <string>Move Directory</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>Extract</string>
   </property>
  </action>
  <action name="actionExtract_to">
   <property name="text">
    <string>Extract to</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableView_1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
